src/osdep/win32/overrides.c

```diff
@@ -679,6 +685,8 @@ int win32_select (int num_fds, struct fd_set *rd, struct fd_set *wr,
 				expired = TRUE;
 		}
 
+		/* Lower CPU Usage WIN64 */
+		Sleep (1);
```

Comment:
I guess the order of instructions can be swapped, so if there is break, no need to sleep.

Reply:
I'm unsure what to do here. Withkuy it the cpu was constantly on near 100%.

Reply:
In select_one_loop there is:
                        /* A Winsock socket */
                        fd_set sock_rd, sock_wr, sock_ex;
                        int    sel_rc;

                        FD_ZERO(&sock_rd);
                        FD_ZERO(&sock_wr);
                        FD_ZERO(&sock_ex);

                        FD_SET(fd - SOCK_SHIFT, &sock_rd);
                        FD_SET(fd - SOCK_SHIFT, &sock_wr);
                        FD_SET(fd - SOCK_SHIFT, &sock_ex);

                        sel_rc = select(fd + 1, &sock_rd, NULL, NULL, &tv);

I think that here can be fd - SOCK_SHIFT + 1 and below too.

BTW, do you know why read, write and exceptions are checked separately?

Reply:
Not really sure what do You mean. Currently it does work. With the fd - SOCK_SHIFT. The select is untouched and doesn't do any troubles. However probably I understand like look at the next file descriptor.
The core is here like this is does work alright. The latest commit I just pushed does get both https and http page on win64. And the cpu consumption is as low as 3-5 percent in idle state.
When I put there Sleep(0) the cpu consumption gets to 100% in idle and stays there.

Reply:
No I get You... I'll retry with the +1 FE_SET.

Reply:
Nope, that didn't help. I tried the fd - SOCK_SHIFT + 1 and CPU is still on 100% ... I spend several hours on this. And the sleep 1 doesn't give any performance  slowdown but get cpu to 1-5% in idle.